AssetLoader.cs

```diff
@@ -185,11 +186,11 @@ public static Texture2D TextureFromImage(Image image)
         {
             try
             {
-                using (MemoryStream stream = new MemoryStream())
+                using (image)
```

Comment:
Not sure if I'm missing something here, what is the point of having a `using` statement here for the image that has been given to us as a parameter?

Reply:
I don't think we should call the Dispose method of the incoming object if it is not necessary.
They should be called at the caller's discretion.

Reply:
xna-cncnet-client was not able to dispose it itself after passing the image, I refactored the code.